lua/lazydocker/config.lua

```diff
@@ -2,7 +2,9 @@ local constants = require("lazydocker.utils.constants")
 local state = require("lazydocker.state")
 
 local M = {
-	__DEFAULT_OPTIONS = {},
+	__DEFAULT_OPTIONS = {
+		border = "double",
+	},
```

Comment:
Might be useful to either point users to a doc page with possible options for border or use enum or whatever to make it clear which values are valid.